### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

**Comment by sschuberth** [comment]
FYI, I believe there was an issue with the coroutines upgrade which made it unusable for us.

**Comment by fviernau** [comment]
> FYI, I believe there was an issue with the coroutines upgrade which made it unusable for us.

Thanks @sschuberth ! I've removed that commit.

File: analyzer/src/main/kotlin/managers/Pip.kt [hunk 1]
```diff
@@ -175,6 +175,7 @@ class Pip(
 
         // TODO: Need to replace this hard-coded list of domains with e.g. a command line option.
         private val TRUSTED_HOSTS = listOf(
+            "repo.platform.here.com",
```
Comment (mnonnenmacher) [inline analyzer/src/main/kotlin/managers/Pip.kt:178]: That seems unrelated.

Comment (fviernau) [reply analyzer/src/main/kotlin/managers/Pip.kt]: Fixed.

File: reporter/src/funTest/kotlin/reporters/CycloneDxReporterTest.kt [hunk 2]
```diff
@@ -43,11 +44,13 @@ class CycloneDxReporterTest : WordSpec({
     val options = mapOf("single.bom" to "true")
 
     "A generated BOM" should {
-        "be valid" {
+        "be valid according to schema version 1.1" {
             val outputDir = createTempDir(ORT_NAME, javaClass.simpleName).apply { deleteOnExit() }
             val bomFile = CycloneDxReporter().generateReport(ReporterInput(ORT_RESULT), outputDir, options).single()
 
-            BomParser().validate(bomFile) should beEmpty()
+            println(bomFile.readText())
```
Comment (mnonnenmacher) [inline reporter/src/funTest/kotlin/reporters/CycloneDxReporterTest.kt:51]: Leftover from debugging?

Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/CycloneDxReporterTest.kt]: Oh soory! Done.
